core/include/kernel/user_access.h

```diff
@@ -34,6 +34,20 @@ static inline TEE_Result copy_from_user(void *kaddr __unused,
 TEE_Result copy_to_user_private(void *uaddr, const void *kaddr, size_t len);
 TEE_Result copy_to_user(void *uaddr, const void *kaddr, size_t len);
 
+TEE_Result clear_user(void *uaddr, size_t n);
```

Comment:
Add an inline fucntion so one get the diff versus a simple `memset(uaddr, 0, n)` instruction.

Reply:
Discard my comment. Looking at the other function below and above this line, I guess it not needed to add a description specifically for this function. Sorry for the noise.